README.md

```diff
@@ -1,4 +1,12 @@
 # prototype_backend
 Barebones Django backend for testing/prototype day
+https://docs.google.com/document/d/1QlrTL4RmjtkbAHtQMJpYgjWWH0QJTh17e7D0h6T35kw/edit
 
-pip3 install django-rest-framework
+pip3 install django-rest-framework django-cors-headers
```

Comment:
Add these to `requirements.txt`. You should be able to `pip install -r requirements.txt` on any branch to install deps, whether that's main or a feature branch.

Reply:
i also realized how this repo has been set up now. assuming you do want to keep the `make dep` command, you should be using `pip install -r requirements.txt` as the command with all the relevant dependencies in `requirements.txt`, so external tooling knows what libraries belong to this project.